server/views/pages/reviewsTable.njk

```diff
@@ -17,14 +17,28 @@
     <a href="/select-location">Select another location</a>
   </p>
 
-  <dl class="app-data-list govuk-!-margin-top-6 govuk-!-margin-bottom-8">
-    <dt>Overdue reviews</dt>
-    <dd>
-      {{- overdueCount -}}
-    </dd>
-  </dl>
+  <div class="govuk-grid-row govuk-!-margin-top-8">
+    <div class="govuk-grid-column-full">
+      <h2 class="govuk-heading-l">Overdue reviews</h2>
```

Comment:
the h2 doesn't need to be in a row as it is allowed to go full width, but doesn't really matter